docs/src/api/class-clock.md

```diff
@@ -249,7 +249,9 @@ Time to be set.
 ## async method: Clock.setSystemTime
 * since: v1.45
 
-Sets current system time but does not trigger any timers.
+Sets system time, but does not trigger any timers. Use this to test how the web page reacts to a time shift, for example switching from summer to winter time, or changing time zones.
+
+Don't use together with [`method: Clock.install`]. Read docs on [clock emulation](../clock.md) to learn more.
```

Comment:
- This should be a part of `setFixedTime`, not `setSystemTime`.

```suggestion
Use this method for simple scenarios where you only need to test with a predefined time. For more advanced scenarios, use [`method: Clock.install`] instead. Read docs on [clock emulation](../clock.md) to learn more.
```

Reply:
moved it in https://github.com/microsoft/playwright/pull/32901/commits/161070bc1b2b9c818c5f42999651dff1b54b682b